src/platform/meteorlake/include/platform/lib/memory.h

```diff
@@ -48,9 +48,10 @@
 #define PLATFORM_HEAP_SYSTEM_SHARED	1
 #define PLATFORM_HEAP_BUFFER		2
 
-/* boot loader in IMR */
-#define IMR_BOOT_LDR_MANIFEST_BASE	0xA1042000
-#define IMR_BOOT_LDR_MANIFEST_SIZE	0x6000
+/**
+ * size of HPSRAM system heap
+ */
+#define HEAPMEM_SIZE 0x40000
```

Comment:
I was considering to submit this separate (remove IMR_BOOT), but it seems you have it here. I guess works like this as well, but we'll have build warnings upon next Zephyr baseline update as long as we have the define in memory.h, so in general better to do those changes in SOF upfront that can be done.

Reply:
here HP heap size is 256KB, how to align this with print out 1362KB total with zephyr build?